categories.yaml

```diff
@@ -5,6 +5,9 @@ classical:
 data-analysis:
     description: Apps for analyzing calculations or experimental data.
     title: Data Analysis
+data-visualization:
+    description: Apps to visualize experimental data
```

Comment:
Following the entry "data-analysis" above, I would replace by "Apps for visualizing experimental data.".